eth/backend.go

```diff
@@ -319,6 +321,11 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	stack.RegisterLifecycle(eth)
 	stack.RegisterLifecycle(confidentialStoreEngine)
 
+	// TODO(lthibault):  replace this with a proper CLI flag.
+	if os.Getenv("SUAVE_EXPERIMENTAL_IPFS") != "" {
+		stack.RegisterLifecycle(&offchain.Env{})
```

Comment:
ideally add a log statement here so it's clearly visible on startup that this is indeed enabled